cypress/plugins/index.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
```

Comment:
**question**: is this file necessary or a place holder worth keeping?

Reply:
I'm not sure if this is necessary, but I think that it's worth keeping.